client/superdesk.config.js

```diff
@@ -38,8 +40,10 @@ module.exports = function(grunt) {
             preview: 1,
             swimlane: {columnsLimit: 4},
             editor3: true,
-            editorHighlights: true
+            editorHighlights: true,
+            autorefresh: true,
```

Comment:
imo the name could indicate what is auto refreshed, like monitoringAutorefresh or something

Reply:
autorefreshContent